types/auto/modules/xmpp/JingleSessionPC.d.ts

```diff
@@ -454,6 +454,15 @@ export default class JingleSessionPC extends JingleSession {
      * @private
      */
     private _initiatorRenegotiate;
+    /**
+     * Adds a new track to the peerconnection. This method needs to be called only when a secondary JitsiLocalTrack is
+     * being added to the peerconnection for the first time.
+     *
+     * @param {JitsiLocalTrack} localTrack track to be added to the peer connection.
+     * @returns {Promise<void>} that resolves when the track is successfully added to the peerconnection, rejected
+     * otherwise.
+     */
+    addTrack(localTrack: any): Promise<void>;
```

Comment:
why `any`? is there no type definition for `JitsiLocalTrack`?

Reply:
same reason as above.

Reply:
then why not declare the type in the method signature? is it a rabbit hole? I think it's good to try adding more strict typing step by step (as long as it doesn't require lots of extra changes which would grow the PR).

Reply:
Type annotations are allowed in the TS files and the original file where the methods is defined is a JS file. Am I missing something ?

Reply:
ah okay I wasn't paying attention to that